torch/_dynamo/variables/nn_module.py

```diff
@@ -241,6 +241,14 @@ def var_getattr(self, tx, name):
 
         return variables.GetAttrVariable(self, name, **options)
 
+    def var_setattr(self, tx, name, val):
+        assert name.is_python_constant() and val.is_python_constant()
+
+        options = VariableTracker.propagate(self)
+        mod = tx.output.get_submodule(self.module_key)
+        setattr(mod, name.as_python_constant(), val.as_python_constant())
```

Comment:
This way doesn't update guards actually, so it will cause correctness issue. I think the right way is to inline ```__setattr__``` function rather than directly call it, since we can't guarantee users don't manipulate tensors in the ```__setattr__``` function.